src/containers/Review/ReviewPageWrapper.tsx

```diff
@@ -2,8 +2,11 @@ import React from 'react'
 import { Container, Form, Header, Label, Message } from 'semantic-ui-react'
 import { Loading, ReviewSection } from '../../components'
 import strings from '../../utils/constants'
+import { ReviewQuestionDecision } from '../../utils/types'
 import useLoadReview from '../../utils/hooks/useLoadReview'
 import { useRouter } from '../../utils/hooks/useRouter'
+import { useUpdateReviewResponseMutation } from '../../utils/generated/graphql'
+import getReviewQuery from '../../utils/graphql/queries/getReview.query'
 
 const ReviewPageWrapper: React.FC = () => {
```

Comment:
Should probably remerge from master and make sure the trigger stuff is still working okay. (It should be, don't you've touched anything here that would affect it)

Reply:
Yes, it's all working. I forgot that TODO in there, but now I changed to just be a comment that make it clear that this check is done inside the `useLoadReview` hook